net/gatling/patches/020-configure_features.patch

```diff
@@ -0,0 +1,22 @@
+Index: gatling-0.16/gatling_features.h
+===================================================================
+--- gatling-0.16.orig/gatling_features.h
++++ gatling-0.16/gatling_features.h
```

Comment:
Git header and commit description, please.

Reply:
The pathing was done with quilt. Should it be done with git instead?

Reply:
Yes please. For other people on rebasing on a new package version it is not bad if there is a clean commit description. What this patch is for.

Reply:
Okay. I added one in the recent version of this PR. Thank you for the advice.